modules/nw-dual-stack-convert.adoc

```diff
@@ -88,13 +90,14 @@ network.config.openshift.io/cluster patched
   path: /spec/platformSpec/baremetal/ingressIPs/-
   value: fd2e:6f44:5dd8::5
 ----
+--
 <1> Ensure that you specify an address block for the `machineNetwork` network where your machines operate. You must select both API and Ingress IP addresses for the machine network.
 <2> Ensure that you specify each file path according to your platform. The example demonstrates a file path on a bare-metal platform.
-
-
-. Patch the infrastructure by entering the following command in your CLI:
+--
 +
-[source,terminal,subs="+quotes,"]
+.. Patch the infrastructure by entering the following command in your CLI:
```

Comment:
Starting with this step, the rest of the procedure isn't rendering correctly in the preview. Probably due to the _..Specify_ error above

Reply:
:+1: